src/api/FlooqApi.sln.DotSettings.user

```diff
@@ -1,4 +1,9 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
 	<s:String x:Key="/Default/Environment/UnitTesting/UnitTestSessionStore/Sessions/=15e2b3a5_002D6b60_002D4bcc_002Dba44_002D6077dcc6bf8c/@EntryIndexedValue">&lt;SessionState ContinuousTestingMode="0" IsActive="True" Name="All tests from &amp;lt;Flooq.Test&amp;gt;" xmlns="urn:schemas-jetbrains-com:jetbrains-ut-session"&gt;&#xD;
-  &lt;Project Location="C:\Users\flums\Documents\Dev\Git\flooq\src\api\Flooq.Test" Presentation="&amp;lt;Flooq.Test&amp;gt;" /&gt;&#xD;
-&lt;/SessionState&gt;</s:String></wpf:ResourceDictionary>
\ No newline at end of file
+  &lt;TestAncestor&gt;&#xD;
+    &lt;TestId&gt;MSTest::50E0EDC9-2E40-4434-8596-95FA85F81FFF::net6.0::Flooq.IntegrationTest.VersionTest.CanGetVersion&lt;/TestId&gt;&#xD;
+    &lt;TestId&gt;MSTest::50E0EDC9-2E40-4434-8596-95FA85F81FFF::net6.0::Flooq.IntegrationTest.DataFlowTest.CanGetDataFlows&lt;/TestId&gt;&#xD;
+  &lt;/TestAncestor&gt;&#xD;
+&lt;/SessionState&gt;</s:String>
+	
+	</wpf:ResourceDictionary>
```

Comment:
What is this file used for? Do we need it?

Reply:
Don't know, maybe included because @silvan-flum and I chose Rider for IDE. Can't remember including this myself.

Reply:
Wrong again. I seem to have included this by mistake when I once tried to open the solution in Visual Studio. I'm going to delete it.